packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -41,7 +41,7 @@ impl RoutedResource for DataTypeResource {
     fn routes<P: GraphPool>() -> Router {
         // TODO: The URL format here is preliminary and will have to change.
         Router::new().nest(
-            "/data-type",
+            "/data-types",
             Router::new()
                 .route("/", post(create_data_type::<P>).put(update_data_type::<P>))
```

Comment:
Do we want to update the handler function names as well?

Reply:
I don't think so, as we still acting on a single data type. I expect to have `get_data_types` or `get_data_type_many` in a follow up

Reply:
Oh yeah true, good point!